views/components/Sidebar.jsx

```diff
@@ -14,8 +18,20 @@ const Sidebar = ({
   totalSubjects,
 }) => {
   const { classes, theme } = useContext(ThemeContext)
-  const [user] = useContext(AuthContext)
+  const [user, setUser] = useContext(AuthContext)
   const avatar = getAvatar({ user })
+  const navigate = useNavigate()
+
+  const handleLogout = async () => {
+    try {
+      await api.auth.logout()
+
+      setUser(null)
+      navigate(routes.login)
+    } catch (error) {
+      throw new Error(error)
```

Comment:
We are catching the error and then throwing a new error wrapping the error. This is almost the same as not catching at all, except not we have an error in an error being thrown. Let's either catch the error and handle it or not catch it at all.

```suggestion
      alert(error.message)
```

Reply:
[fd6e0cb](https://github.com/AMP-SCZ/dpdash/pull/323/commits/fd6e0cbe82ffb8f78b65b8841e1d9e4dd676d278)